src/Cake.Curl/ArgumentsExtensions.cs

```diff
@@ -77,6 +78,20 @@ internal static class ArgumentsExtensions
             {
                 arguments.Append("--retry-connrefused");
             }
+
+            if (settings.MaxTime.HasValue)
+            {
+                arguments.AppendSwitch(
+                    "--max-time",
+                    settings.MaxTime.Value.TotalSeconds.ToString(CultureInfo.CurrentCulture));
+            }
+
+            if (settings.ConnectionTimeout.HasValue)
+            {
+                arguments.AppendSwitch(
+                    "--connect-timeout",
+                    settings.ConnectionTimeout.Value.TotalSeconds.ToString(CultureInfo.CurrentCulture));
```

Comment:
Here, we can simply say `settings.ConnectionTimeout.Value.TotalSeconds` since the compiler will add the call to `ToString()` when using [string interpolation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated). Also, when invoking `ToString()` on a `Double` the usage of the current culture is implicit.

Reply:
@ecampidoglio  See [comment](https://github.com/ecampidoglio/Cake.Curl/pull/27#discussion_r270528142).

Reply:
@ecampidoglio Correct — it seems I was a little too quick commenting here. 😳
I don't see the value in using string interpolation if there are no string literals involved. It would be enough to simply say `settings.ConnectionTimeout.Value.TotalSeconds.ToString()`.

Reply:
@deqenq See [this comment](https://github.com/ecampidoglio/Cake.Curl/pull/27#discussion_r271191018).